allure-generator/package.json

```diff
@@ -68,8 +68,7 @@
     "font-awesome": "^4.5.0",
     "handlebars": "^4.0.5",
     "highlight.js": "^9.9.0",
-    "i18next": "^7.1.0",
-    "i18next-intervalplural-postprocessor": "^1.0.0",
```

Comment:
How come that you have removed that? It was used to generate 1 item/N items pluralization